h2o-core/src/test/java/water/rapids/RapidsTest.java

```diff
@@ -629,6 +631,53 @@ static boolean checkSaneFrame_impl() {
     }
   }
 
+  @Test public void testNaOmit() {  // code from Nidhi
```

Comment:
Yeah, new test AstNaOmitTest.java is added to rapids/ast.prims/mungers directory.